### DISCUSSION TITLE
Allow offline reading of posts

### DISCUSSION THREAD

**Discussion by JacekAndrzejewski** [forem/forem#19179] [Feature Requests]
Currently whenever post is loaded it's enough to turn off mobile data and Wi-Fi to no longer see it, but rather see a huge icon and notification that network is not available.
It would be great to have possibility to read already loaded post, or maybe have a way of downloading posts for offline reading (compare Medium which allows downloading reading lists and bunch of other software probably)

**Comment by rc2168st** [comment]
high priority
